### DISCUSSION TITLE
布告板集合图片变黑

### DISCUSSION THREAD

**Discussion by HU-Wenxing-ZuiShuai** [zouyaoji/vue-cesium#384] [Q&A]
使用布告板集合时，使用多张图片后，该集合内的所有图片变成黑色的，具体情况如下：使用一张或者两到三张图片可正常显示，我这边是有十张图片，进入页面后会正常加载一下，然后马上集合内的所有图片都变成黑色的了

**Comment by MangMax** [comment]
我也遇到类似的问题，我是文字渲染cavans图像的。第一次加载正常，重载组件后，甚至是重载其他组件布告板就会变黑，

**Reply by zouyaoji** [reply]
哈喽，你们能不能确定是vue-cesium的bug还是cesium的bug，
如果是vue-cesium的bug，可以提供点例子，我空了瞧瞧。

**Reply by HU-Wenxing-ZuiShuai** [reply]
暂时不能确定时哪个的问题，例子如下，图片我这边是10张不同的`<vc-collection-billboard ref="rainStCollectionRef" :show="rainStvisible">
        <vc-billboard
          v-for="item in rainStationArr"
          :position="item.coord"
          :distance-display-condition="[0, 2000000]"
          :scale="0.04"
          :height-reference="2"
          :id="item.stcd"
          :image="nginxUrl +item.img"
          :color="item.color"
        ></vc-billboard>
      </vc-collection-billboard>`
